build.gradle

```diff
@@ -13,12 +13,11 @@ targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
 
```

Comment:
implementation 'org.apache.httpcomponents:httpclient:4.5.13' should be from the  dependencies.